app/views/reservations/_open_hours.html.haml

```diff
@@ -0,0 +1,23 @@
+.hidden-phone
```

Comment:
I feel we have a lot of repeated code in this file but I don't see an obvious way of avoiding repetition. Leaving a comment in case the next reviewer can think of something

Reply:
The other option is to do something like this https://stackoverflow.com/a/53788026 although the result is not exactly the same with bootstrap styles

Reply:
i ended up doing what's on the stackoverflow post

Reply:
And now I reverted because it does not look good in safari nor firefox 🥲